mediaqueries/Overview.bs

```diff
@@ -1607,7 +1607,7 @@ Pointing Device Quality: the 'pointer' feature</h3>
 	Type: discrete
 	</pre>
 
-	The 'pointer' media feature is used to query about the presence and accuracy of a pointing device such as a mouse.
+	The 'pointer' media feature is used to query the presence and accuracy of a pointing device such as a mouse.
```

Comment:
Is this to fix incorrect grammar, or are both sentences valid but mean different things? My non native speaker brain prefers the current phrasing, but I am very possibly either missing a nuance or not noticing a mistake.